Gemfile.lock

```diff
@@ -249,25 +249,25 @@ GEM
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
-    net-ssh (6.0.2)
+    net-ssh (6.1.0)
     netrc (0.11.0)
     nio4r (2.5.4)
-    nokogiri (1.10.9)
+    nokogiri (1.10.10)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.6)
+    oj (3.10.14)
```

Comment:
### [oj](http://www.ohler.com/oj/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.10.6 → 3.10.14

[_[change-log](https://github.com/ohler55/oj/blob/master/CHANGELOG.md), [source-code](https://github.com/ohler55/oj)_]





<details>
<summary>Commits</summary>

A change of **33** commits. See the full changes on [the compare page](https://github.com/ohler55/oj/compare/v3.10.6...v3.10.14).

These are the first 10 commits:
- (14f9770) [Improve error message](https://github.com/ohler55/oj/commit/14f97706cf0b5a411f7df733e45868504a66efe9)
- (a253525) [chore: run on latest Ruby runtimes (#599)](https://github.com/ohler55/oj/commit/a2535250511fd238437844246e181431b882c73a)
- (6d99640) [Faster float handling](https://github.com/ohler55/oj/commit/6d99640fa160fda801e3e0a8aeda8827429d5590)
- (e8c241d) [Faster float handling](https://github.com/ohler55/oj/commit/e8c241d87e034dcf1aba14eec473f4d55e208071)
- (c8103c3) [Fix rounding](https://github.com/ohler55/oj/commit/c8103c3639b477aca61c1f21f760762808aa43b4)
- (a37b495) [English fixes for Options.md (#606)](https://github.com/ohler55/oj/commit/a37b4956482d5f699f4f9b79e49307b93331c882)
- (80430c5) [More twists and turns for floats](https://github.com/ohler55/oj/commit/80430c5404a0af21e785e72aa0a5a8f8a5c371d6)
- (1a56feb) [Merge branch 'faster-floats' into develop](https://github.com/ohler55/oj/commit/1a56febf7ac5d778766a0f4a591efa4adde6cb7a)
- (182fdb9) [Release prep](https://github.com/ohler55/oj/commit/182fdb9324c535b5c332785396f6213565100a6e)
- (ee1fa2e) [Revert to a simplier float converter](https://github.com/ohler55/oj/commit/ee1fa2ea6c0304557b6fe576ae172fb429eb7247)


</details>